packages/data-stores/src/i18n/test/resolvers.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Internal dependencies
+ */
+import { getLocalizedLanguageNames } from '../resolvers';
+import { setLocalizedLanguageNames } from '../actions';
+
+describe( 'i18n resolvers', () => {
+	describe( 'getLocalizedLanguageNames', () => {
+		it( 'should set the localized language names', () => {
+			const iter = getLocalizedLanguageNames( 'en-us' );
+
+			expect( iter.next().value ).toEqual( {
+				type: 'API_FETCH',
+				request: expect.objectContaining( {
+					mode: 'cors',
+					url: expect.stringContaining( 'language-names?_locale=en-us' ),
+				} ),
+			} );
+
+			const localizedLanguageNames = Symbol( 'localizedLanguageNames' );
+
+			expect( iter.next( localizedLanguageNames ).value ).toEqual(
```

Comment:
Should we add one last test case here to ensure that after this one, in the last fulfilment `done` is `true`?

Reply:
Wouldn't that just lock the function down needlessly? For the purposes of this test, what does it care if something additional is added or done?

In fact, these tests are already so flimsy. If someone decides to add a middle step between API Fetch and setting the localized language names, then they'll need to update the test to reflect that new ordering... simply by adding the exact same call they made in the code to the test itself. That seems silly to me. I'm not sure these tests are useful at all.

Reply:
FWIW, there are a bunch of those tests in Gutenberg for where action generators are being used. I find them pretty useful, to be honest, for two reasons:

* Generators aren't just functions, so sometimes understanding them can be difficult from a first read. Reading the tests, though, makes that way easier.
* Generators can end up being complex sometimes, and verifying that their execution flow produces the same results and yields the same output is crucial for ensuring we're not breaking them as we iterate. 

Anyway, as pointed out before, I don't consider this a blocker for the PR and I'm happy to accept whatever you prefer to go ahead with here.